namespaces-allowlist.yaml

```diff
@@ -6,3 +6,4 @@ firefox_ios:
 fenix:
   owners:
     - vzare@mozilla.com
+bergamot: null
```

Comment:
Should this have `owners`?

Reply:
`owners` is optional, and simply appends additional owners to the ones from the probe-info-service. If they need more we can add them later.